ajax/libs/vast-client/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "vast-client",
+  "filename": "vast-client.min.js",
+  "description": "Javascript VAST Client",
+  "version": "1.8.0",
+  "keywords": [
+    "vast",
+    "ad",
+    "advertising",
+    "iab",
+    "in-stream",
+    "video"
+  ],
+  "author": "Olivier Poitrey <rs@dailymotion.com>",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/dailymotion/vast-client-js"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/dailymotion/vast-client-js.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "*.(js|map)"
```

Comment:
This pattern can't get any files.
Let's modify to `"vast-client*"`.
PS. You could try auto-update tool to check.

Many thanks.